indico/web/forms/fields.py

```diff
@@ -774,7 +778,13 @@ def _value(self):
 
 
 class IndicoDateTimeField(DateTimeField):
-    """"Friendly datetime field that handles timezones and validations."""
+    """Friendly datetime field that handles timezones and validations.
+
+    Important: When the form has a `timezone` field it must be
```

Comment:
No, WTForms iterates over the form's field list and processes them in that order:

```python
        for name, field, in iteritems(self._fields):
            if obj is not None and hasattr(obj, name):
                field.process(formdata, getattr(obj, name))
            elif name in kwargs:
                field.process(formdata, kwargs[name])
            else:
                field.process(formdata)
```